Front-End/challenge2/script.js

```diff
@@ -5,3 +5,171 @@ TO-DO:
 - The calculator should be completely functional
 
 */
+
+let a;
+let b;
+let operation;
+let total = 0;
+let prev = false;
+let fromEqual = false;
+
+const display = document.getElementById("display");
+const zero = document.getElementById("zero");
```

Comment:
Do you think is a better way to get the values? I'd like you to think how you can avoid adding a selector for each number...let's say for example that you have 100 numbers? how would you that? is there another selector you can use? maybe one selector for all numbers and another for operations? try to think of a better way to avoid duplicating this